library/Sexp/src/Juvix/Sexp/Structure/Frontend.hs

```diff
@@ -225,6 +230,15 @@ data DefHandler = DefHandler
 -- Not Generated, due to limited generation
 -------------------------------------------
 
+--------------------
+-- NotPunned no Grouping
+--------------------
+fromNotPunnedGroup :: [NotPunned] -> Sexp.T
+fromNotPunnedGroup = Sexp.unGroupBy2 . toStarList fromNotPunned
+
```

Comment:
Would it make sense to have a `NotPunned with Grouping` comment here, since there's a `no Grouping` one above?